src/CommandRunner.cpp

```diff
@@ -235,13 +235,13 @@ Try<Nothing> runCommandWithTimeout(const std::string& command,
         exitCode = WEXITSTATUS(status);
         TASK_LOG(ERROR, loggingMetadata)
             << "Failed to successfully run the command \"" << command
-            << "\", it failed with status " + std::to_string(exitCode);
+            << "\", it failed with status " + std::to_string(exitCode) << ".";
```

Comment:
Should use the `<<` operator instead of `+`

Reply:
You're right! Thanks for noticing!

Reply:
Fixed